### DISCUSSION TITLE
Main window is hidden on macOS

### DISCUSSION THREAD

**Discussion by ghost** [keepassxreboot/keepassxc#7355] [Q&A]
My KeepassXC database opens fine. But after a while when I try to switch back to KeepassXC, I can't see the KeepassXC main window anymore. Although, I can see all the menubar items and able to click the menubar items drop-down list. I can only see the four corners of the main window.

This has started shortly after I installed KeepassXC (May, 2021). I tried the following to no avail:

- installed the beta version 2.7 in November, 2021 (https://snapshot.keepassxc.org/latest/)
- removed all PDF attachments from KeepassXC and reduced the size of the database to 20MB
- changed the desktop wallpaper (to see if the main window is visible)

Running KeepassXC 2.6.6 on Macbook Pro 2015 with 8 GB RAM. Database size is 32 MB. Using it with Firefox 96 with KeepassXC-Browser plugin. OS: MacOS Monterey 12.1.

Thanks.

**Comment by droidmonkey** [comment]
Sounds like the window is hidden, not crashed.

**Reply by ghost** [reply]
> Sounds like the window is hidden, not crashed.

Yes, that is correct.

Is there a way to resolve this issue? Thanks.

**Reply by droidmonkey** [reply]
Never encountered it before. Are you sure it isn't just hidden to the tray icon?

**Reply by ghost** [reply]
> Never encountered it before. Are you sure it isn't just hidden to the tray icon?

No, it isn't. I tried toggling hide/ unhide with no luck.

**Reply by phoerious** [reply]
Stop the KeePassXC process, delete `~/Library/Application\ Support/KeePassXC/keepassxc.ini` and then try again.

**Reply by ghost** [reply]
> Stop the KeePassXC process, delete `~/Library/Application\ Support/KeePassXC/keepassxc.ini` and then try again.

Thanks. This seems to have resolved the issue. The only change I made after doing the above is browser integration. I haven't encountered the issue in the last couple of days.
